src/components/toggle/toggle.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Toggle from './index';
+import { setup } from '../../utils/state.mock';
+
+describe('Toggle', () => {
+  const input = (wrapper) => wrapper.find('.pipeline-toggle-input');
+  const label = (wrapper) => wrapper.find('.pipeline-toggle-label');
+
+  it('is checked when checked is true', () => {
+    const wrapper = setup.mount(
+      <Toggle checked={true} onChange={jest.fn()}>
+        {' '}
+      </Toggle>
+    );
+    expect(input(wrapper).prop('checked')).toBe(true);
```

Comment:
I'm not entirely sure about the effectiveness of this test given this is technically testing the functionality of react in passing props which isn't needed - this could be deleted.

Reply:
@studioswong  -- i thought it was doing that. I am confused now, can we have a quick 1:1 where you can explain to me this further. thanks

Reply:
resolving this as per our discussion in our 1:1. Thanks @rashidakanchwala